server/db/connection.js

```diff
@@ -0,0 +1,14 @@
+import { Pool } from 'pg';
+import dotenv from 'dotenv';
+
+
+dotenv.config();
+
+
+const connectionString = process.env.DATABASE_URL;
```

Comment:
Please add this environment variable to your .env.sample file. and also this detail in your README file under the installing section @Bobjayafam